src/hotspot/share/jfr/recorder/storage/jfrVirtualMemory.cpp

```diff
@@ -104,7 +104,7 @@ bool JfrVirtualMemorySegment::initialize(size_t reservation_size_request_bytes)
   assert(is_aligned(reservation_size_request_bytes, os::vm_allocation_granularity()), "invariant");
   _rs = ReservedSpace(reservation_size_request_bytes,
                       os::vm_allocation_granularity(),
-                      os::vm_page_size());
+                      os::vm_page_size(), mtTracing);
```

Comment:
The mtTracing should probably be on a separate line, so that it follows the style of the surrounding code.

Reply:
Done.